es_ES/flutter-for-ios.md

```diff
@@ -0,0 +1,1937 @@
+        appBar: AppBar(
+          title: Text("Sample App"),
+        ),
+        body: getBody());
+  }
+
+  ListView getListView() => ListView.builder(
+      itemCount: widgets.length,
+      itemBuilder: (BuildContext context, int position) {
+        return getRow(position);
+      });
+
+  Widget getRow(int i) {
+    return Padding(padding: EdgeInsets.all(10.0), child: Text("Row ${widgets[i]["title"]}"));
+  }
+
+  loadData() async {
+    ReceivePort receivePort = ReceivePort();
+    await Isolate.spawn(dataLoader, receivePort.sendPort);
+
+    // El 'echo' isolate envía su SendPort como primer mensaje
+    SendPort sendPort = await receivePort.first;
+
+    List msg = await sendReceive(sendPort, "https://jsonplaceholder.typicode.com/posts");
+
+    setState(() {
+      widgets = msg;
+    });
+  }
+
+// El punto de entrada para el aislamiento
+  static dataLoader(SendPort sendPort) async {
+    // Para los mensajes entrantes, abra el ReceivePort.
+    ReceivePort port = ReceivePort();
+
+    // Notifique a cualquier otro aislado a qué puerto escucha este aislado.
+    sendPort.send(port.sendPort);
+
+    await for (var msg in port) {
+      String data = msg[0];
+      SendPort replyTo = msg[1];
+
+      String dataURL = data;
+      http.Response response = await http.get(dataURL);
+      // Lotes de JSON para analizar
+      replyTo.send(json.decode(response.body));
+    }
+  }
+
+  Future sendReceive(SendPort port, msg) {
+    ReceivePort response = ReceivePort();
+    port.send([msg, response.sendPort]);
+    return response.first;
+  }
+}
+{% endprettify %}
+
+## Como puedo hacer peticiones de red?
+
+Realizar una petición de red en Flutter es fácil cuando se utiliza la popular función
+[`http` package](https://pub.dartlang.org/packages/http). Esto abstrae una gran parte de la red que normalmente podría implementar usted mismo,
+lo que simplifica la realización de peticiones de red.
+
+Para utilizar `http` package, añadirlo a sus dependencias en `pubspec.yaml`:
+
+<!-- skip -->
+{% prettify yaml %}
+dependencies:
+  ...
+  http: ^0.11.3+16
+{% endprettify %}
+
+Para realizar una llamada de red, llame `await` en la `async` función `http.get()`:
+
+<!-- skip -->
+{% prettify dart %}
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+import 'package:http/http.dart' as http;
+[...]
+  loadData() async {
+    String dataURL = "https://jsonplaceholder.typicode.com/posts";
+    http.Response response = await http.get(dataURL);
+    setState(() {
+      widgets = json.decode(response.body);
+    });
+  }
+}
+{% endprettify %}
+
+## ¿Cómo puedo mostrar el progreso de una tarea de larga duración?
+
+En iOS, normalmente se utiliza un `UIProgressView` durante la ejecución de un
+una larga tarea en segundo plano.
+
+En Flutter, utilice un `ProgressIndicator` widget.
+Muestre el progreso programáticamente controlando cuándo se renderiza
+a través de una señal booleana. Dile a Flutter que actualice su estado antes de que tu
```

Comment:
Tell Flutter to update its state before your long-running task starts, and hide it after it ends.
Dile a Flutter que actualice este estado antes de comenzar la tarea de larga duración, y ocúltalo después de que acabe.